charts/steward/values.yaml

```diff
@@ -42,16 +43,18 @@ tenantController:
       cpu: 10m
   podSecurityContext: {}
     # fsGroup: 2000
-  securityContext: {}
-    # capabilities:
-    #   drop:
-    #   - ALL
-    # readOnlyRootFilesystem: true
-    # runAsNonRoot: true
-    # runAsUser: 1000
+  securityContext:
+    capabilities:
+      drop:
+      - ALL
+    readOnlyRootFilesystem: true
+    runAsNonRoot: true
+    runAsUser: 1000
+    runAsGroup: 1000
   nodeSelector: {}
   affinity: {}
   tolerations: []
+  possibleTenantRoles: ["steward-tenant"]
```

Comment:
Didn't dig deeper... is `steward-tenant` correct?

Reply:
> Didn't dig deeper... is `steward-tenant` correct?

Yes, it is. This is the role we bind within a tenant namespace to the client service account and the tenant service account.

Reply:
We use a chart value here because administrators can bring their own role (configured as annotation of the client namespace).